x/inflation/types/v1beta2/inflation_calculator.go

```diff
@@ -0,0 +1,68 @@
+package v1beta2
+
+import (
+	"math"
+	"strconv"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var (
+	// these are set during NewApp and used in InflationCalculator
+
+	GenesisTime            time.Time
+	InflationParamSubspace paramstypes.Subspace
+)
+
+func InflationCalculator(ctx sdk.Context, minter minttypes.Minter, params minttypes.Params, bondedRatio sdk.Dec) sdk.Dec {
+	var inflationParams Params
+	InflationParamSubspace.GetParamSet(ctx, &inflationParams)
+	tHalf := float64(inflationParams.InflationDecayFactor)
+	initialInflation, err := strconv.ParseFloat(inflationParams.InitialInflation, 32)
```

Comment:
Params should be represented by the appropriate type here.  See how `x/deploy` does it [here](https://github.com/ovrclk/akash/blob/9dbc53f3b4d51025534228817bdbe791f3b0a1a9/x/deployment/types/v1beta2/params.go).

I agree with @hydrogen18  - we should use a decimal or rational here.

Reply:
Done